System/AbstractFilePath/Data/ByteString/Short/Internal.hs

```diff
@@ -0,0 +1,424 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE MagicHash #-}
+{-# LANGUAGE PackageImports #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE UnboxedTuples #-}
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE ViewPatterns #-}
+
+module System.AbstractFilePath.Data.ByteString.Short.Internal where
```

Comment:
Can you add a module header, explaining what's this module about? It's difficult to reason in an absense of explicit export list.